src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerIntellisenseFilter.cs

```diff
@@ -120,24 +106,20 @@ void executeNextCommandTarget()
                 // legacy stuff they interfaced with. That means we get SCROLLUP if the user
                 // types escape, so treat SCROLLUP like CANCEL. It's actually a CANCEL.
                 case VSConstants.VSStd2KCmdID.SCROLLUP:
```

Comment:
I believe Dustin Coleman said this is no longer needed?

Reply:
@jasonmalinowski I could not see this code used in threads I reviewed at #32631. 
So, I would get an confirmation from Dustin Coleman.

Reply:
It's possible it can be removed too. I was somewhat limiting this PR to code that was 100% clearly dead.

Reply:
(This was deleted as a part of another pull request.)